lib/docs/scrapers/composer.rb

```diff
@@ -0,0 +1,27 @@
+module Docs
+  class Composer < UrlScraper
+    self.name = 'Composer'
+    self.type = 'composer'
```

Comment:
I think you can set this to `'simple'` if you don’t have any special styles.

Reply:
Done!